src/3box.js

```diff
@@ -86,7 +86,7 @@ class Box extends BoxApi {
     const address = await this._3id.getAddress()
     const rootstoreAddress = address ? await this._getRootstore(address) : null
     if (rootstoreAddress) {
-      await this.replicator.start(rootstoreAddress, { profile: true })
+      await this.replicator.start(rootstoreAddress, this._3id.DID, { profile: true })
```

Comment:
This won't work since the 3ID won't be authenticated. We should probably create a method for setting the DID and call it after line 100 where `setIdentity` is being called.